docs/README.md

```diff
@@ -1756,7 +1756,7 @@ Otherwise, port forward to the kiali pod directly:
 kubectl port-forward -n istio-system svc/kiali 20001:20001
 ```
 
-You can view Kiali dashboard at: http://localhost:20001
```

Comment:
actually, this change will have to be made in the community repo - otherwise we'll have merge conflicts

Reply:
Oh, OK.  Shall I close this and open one upstream?

Reply:
I'm afraid no. You just have to remove the change in `docs/README.md` file from this PR. The changes to the file `docs/ossm/ossm2-vs-ossm3.md` can be retained in this PR as this file is only present in this repo (and not in upstream).

Reply:
done